### DISCUSSION TITLE
Importing Particles Js Boot Error

### DISCUSSION THREAD

**Discussion by iamshulz** [quasarframework/quasar#9380] [General - Components / Directives / etc]
I am trying to import Particles.js and directly referenced it from `boot` 
quasar.conf.js
```
boot: [
      'axios',
      '~particles.vue3/dist/particles.vue.js'
    ],
 ```
 here is the error
 ```
 client-entry.js?2f39:84 [Quasar] boot error: TypeError: app.component is not a function
    at Array.VueParticles (particles.vue.js?6aae:94)
    at start (client-entry.js?2f39:68)
```
Error happens on particles.vue.js
```
const VueParticles = (app, options) => {
    app.component("Particles", script);
};
```

**Comment by ontwikkelfabriek** [comment]
@jamols09 

That's not the way to use Quasar boot files.


Instead ( using Quasar V2 with Vue3 ):
`src/boot/particles.js`
```javascript
import Particles from "dist/particles.vue3"

export default boot(({ app }) => {
   app.use(Particles)
})
```

**Reply by iamshulz** [reply]
Sorry, I didn't know how to boot a file since the axios has a different setup. I think you probably have interchange the file path `'particles.vue3/dist/particles.vue'`

**Reply by iamshulz** [reply]
> @jamols09
> 
> That's not the way to use Quasar boot files.
> 
> Instead ( using Quasar V2 with Vue3 ):
> `src/boot/particles.js`
> 
> ```js
> import Particles from "dist/particles.vue3"
> 
> export default boot(({ app }) => {
>    app.use(Particles)
> })
> ```

Sorry of the rebump I was able to get it working however the issue came back again I wonder what went wrong?
![image](https://user-images.githubusercontent.com/20100579/119235196-de42ac80-bb63-11eb-9610-ed21db936e3d.png)
